build.gradle

```diff
@@ -19,7 +19,7 @@ repositories {
 }
 
 ext {
-    scalaVersion = "2.11"
+    scalaVersion = "2.12"
```

Comment:
is there a way to parameterize this, so that we don't have branch drift in the meantime while we wait for 2.12 in OW?

Reply:
Can possibly be done but may introduce some complexity. As we may not be needing to support multiple Scala version ... we can keep this simple for now